Wikipedia/Code/WMFLocationManager.m

```diff
@@ -14,8 +14,6 @@
 
 @interface WMFLocationManager () <CLLocationManagerDelegate>
```

Comment:
Commit message:

Remove unused property `significantLocationUpdatesOnly`
This property was never set.